README.md

```diff
@@ -8,7 +8,7 @@ Humidifier allows you to build AWS CloudFormation (CFN) templates programmatical
 
 For the full docs, go to [https://localytics.github.io/humidifier/](http://localytics.github.io/humidifier/). For local development instructions, see the [Development](https://localytics.github.io/humidifier/#label-Development) section.
```

Comment:
It looks like the Development instructions need to be updated to specify Ruby 2.1 as well if they aren't already (big PR)

https://localytics.github.io/humidifier/#label-Development